### PR TITLE
Convert keys to .talon-list where possible

### PR DESCRIPTION
**Pull request by knausj85**
- does not port punctuation_words or symbol_keys, as these are used by create spoken forms
- adds support for keypad keys

### CODE REVIEW

**Comment by knausj85** [comment]
We discussed the old PR (https://github.com/talonhub/community/commit/940fe694bdb1b5ea752fc2e62154a255f055734e) a bit in the backlog session. The thinking is to avoid trying to create some sort of auto-conversion mechanism and live with a manual merge for python lists we convert to *.talon-list.

File: core/keys/function_key.talon-list [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+f four: f4
+f five: f5
+f six: f6
+f seven: f7
+f eight: f8
+f nine: f9
+f ten: f10
+f eleven: f11
+f twelve: f12
+f thirteen: f13
```
Comment (AndreasArvidsson) [inline core/keys/function_key.talon-list:15]: We should probably comment everything above twelve

**Review by AndreasArvidsson — CHANGES_REQUESTED** [review]
We should probably leave a link to the wiki in letters and modifier lists

**Comment by nriley** [comment]
https://talon.wiki/Quickstart/improving_recognition_accuracy#collected-alternatives-to-the-default-alphabet is for the alphabet; we should migrate our recommended modifier alternatives to the wiki too.

**Comment by knausj85** [comment]
Since the modifiers aren't already on the wiki, maybe we save that bit for another. I think this is good to go.

**Review by nriley — CHANGES_REQUESTED** [review]

**Comment by nriley** [comment]
> Since the modifiers aren't already on the wiki, maybe we save that bit for another day. I think this is good to go.

Heh, I didn't see this before I put together a wiki change :-) Let me know if you want some other alternatives; I included the ones from the comments plus the ones I use personally.

https://github.com/TalonCommunity/Wiki/pull/302/files#diff-6eb918b959893584905f667cd9806ebf3f89362a8ecf3534138f985d80236fc2R92

File: core/keys/keys.py [hunk 2]
```diff
@@ -42,6 +36,12 @@ def number_key(m) -> str:
     return m.number_key
 
 
+@mod.capture(rule="{self.keypad_key}")
```
[the remark this answers is not in the document]
Comment (knausj85) [reply core/keys/keys.py:39]: Maybe we open an issue, and do a cleanup pass on this file separately from this PR? Not sure it's worth holding up the PR, and the removal of other captures is a (theoretically) breaking change.

**Comment by knausj85** [comment]
> > Since the modifiers aren't already on the wiki, maybe we save that bit for another day. I think this is good to go.
> 
> Heh, I didn't see this before I put together a wiki change :-) Let me know if you want some other alternatives; I included the ones from the comments plus the ones I use personally.
> 
> https://github.com/TalonCommunity/Wiki/pull/302/files#diff-6eb918b959893584905f667cd9806ebf3f89362a8ecf3534138f985d80236fc2R92

Those look good to me, thanks. Forgot I'm a member over on the wiki org too, and can do things 🐙

File: core/keys/keys.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nriley) [reply core/keys/keys.py:39]: Sure, I'm OK either way!

[the remark this answers is not in the document]
Comment (knausj85) [reply core/keys/keys.py:39]: https://github.com/talonhub/community/issues/1560

**Review by nriley — APPROVED** [review]

**Comment by nriley** [comment]
Ugh, we missed 'insert" too...

**Pull request merged**